LambdaEngine/Include/Threading/API/GenericThread.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include "LambdaEngine.h"
+
+namespace LambdaEngine
+{
+	/*
+	* GenericThread
+	*/
+
+	typedef void* ThreadHandle;
+
+	class GenericThread
+	{
+	public:
+		static ThreadHandle GetCurrentThreadHandle()
+		{
+			return nullptr;
+		}
+
+		static ThreadHandle GetThreadHandle(std::thread& thread)
```

Comment:
What the linter said

Reply:
This is a base implementation, i.e if there is no implementation, this means that it should be the same for all implementations. Therefore we should not mark this as const here since we use the native thread in std::thread in the definitions. The alternative is to just return std:: thread::native_handle here as our ThreadHandle.

Reply:
As you can see here https://en.cppreference.com/w/cpp/thread/thread/native_handle, the native handle is not defined for const std::thread

Reply:
So what do you think, should we define this function like this, or should we move Win32Thread::GetThreadHandle logic to the base?

Reply:
In that case, it's fine to leave it as-is.